src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALLS = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
+            System.out.println(new Lottery().getRandomBall().toString());
+        }
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < NUMBER_OF_BALLS; i++) {
            System.out.println(lottery.getRandomBall().toString());
        }
```
it is better to create a separate object and use it in a loop instead of creating a new one each time